docs: add notes for making pre-releases (#344)
BuildingSync/schema: docs/release_instructions.md

* Checkout develop and pull the most recent changes

* Update the Version in the header of the XSD in three places:
    * Update version the `/xs:schema@version`.
    * Update version in the "schema title", at `/xs:schema/xs:annotation/xs:documentation[1]`.

>     * If creating an official release (ie you are NOT creating a pre-release), add the version as an enumeration to the `auc:BuildingSync` `version` attribute with the latest version. Though we historically added some pre-releases to `@version`, they should no longer be included.

Reviewer: ```suggestion
    * If creating an official release (i.e., you are NOT creating a pre-release), add the version as an enumeration to the `auc:BuildingSync` `version` attribute with the latest version. Though we historically added some pre-releases to `@version`, they should no longer be included.
```

* Update the CHANGELOG.md to include the latest changes, and the most recent version.

> 	* Run the change_log.rb script (e.g. ruby src/change_log.rb -t abcdefghijklmnopqrstuvwxyz -s 2019-12-21). The date range must span from the last official release (ie don't start at a pre-release) until the current date.

Reviewer: ```suggestion
	* Run the change_log.rb script (e.g., ruby src/change_log.rb -t abcdefghijklmnopqrstuvwxyz -s 2019-12-21). The date range must span from the last official release (ie don't start at a pre-release) until the current date.
```

git checkout master && git pull
git tag -a v<version> -m "<message>" [SHA]
```

> Where `v<version>` is a valid [semantic version](https://semver.org/) (e.g. `v1.2.3` or `v1.2.3-pr.1`) and `<message>` is the tagging message (e.g. "First official release"). See [Versioning](versioning.md) for more information.

Reviewer: ```suggestion
Where `v<version>` is a valid [semantic version](https://semver.org/) (e.g., `v1.2.3` or `v1.2.3-pr.1`) and `<message>` is the tagging message (e.g. "First official release"). See [Versioning](versioning.md) for more information.
```
